### PR TITLE
[CROW-201] Update Create project page to match designs

### PR DESCRIPTION
**Pull request by FacuEM**
#### :page_facing_up: Description:

In this PR the Create page is updated to look the same as the designs and to include the new optional fields.

---

#### :pushpin: Notes:

It is not contemplated in this PR:
- Validation for optional fields.
- Back modal.
- Animation on the created-projects page.

---

#### :play_or_pause_button: Demo:
https://www.loom.com/share/3820f0b4dbad4076891e71a9c9f952c1

### CODE REVIEW

File: components/modal/modal.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  return (
+    <ReactModal
+      isOpen={isModalOpen}
+      shouldCloseOnEsc
+      shouldCloseOnOverlayClick
+      onRequestClose={() => {
+        handleModal(false);
+      }}
+      style={{
+        overlay: {
```
Comment (renacargnelutti) [inline components/modal/modal.js:20]: NIT: how about managing this as a `const`?

File: components/modal/modal.js [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+      <Description>
+        The project will be deleted automatically and you will lose your
+        progress.
+      </Description>
+
+      <ButtonsContainer>
+        <Button
+          onClick={() => {
+            handleModal(false);
```
Comment (renacargnelutti) [inline components/modal/modal.js:54]: NIT: how about adding handlers for these events? This may improve code readability I guess

File: package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
     "start": "next start",
     "lint": "next lint",
     "test": "jest",
-    "test:cov": "yarn test --silent --coverage --coverageReporters=text --coverageReporters=text-summary --watchAll=false",
+    "test:cov": "yarn test --silent --coverage --coverageReporters=text --coverageReporters=text-summary --watchAll=false --passWithNoTests",
```
Comment (renacargnelutti) [inline package.json:11]: Out of curiosity, why did you add this option?

File: utils/post.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+      endDate,
+      story,
+      fundGoal,
+      token,
+      url,
+      collaborators,
+    } = formData;
+
+    await axios.post(
+      `${process.env.NEXT_PUBLIC_CROWDFUNDING_API}/campaigns`,
```
Comment (renacargnelutti) [inline utils/post.js:20]: NIT: Probably you can use destructuring like 
```js
const { NEXT_PUBLIC_CROWDFUNDING_API } = process.env;
```

**Review by renacargnelutti — APPROVED** [review]
Looks good to me! I just left some comments to see what you think about that

File: package.json (hunk 3 above)
Comment (FacuEM) [reply package.json:11]: To handle this issue: 
Screenshot 2023-03-22 at 11 35 36

Comment (FacuEM) [reply package.json:11]: Currently there are no tests

Comment (renacargnelutti) [reply package.json:11]: Got it! I guess it's the same as deleting the file.. it's ok for me the way you chose
